via/static/scripts/video_player/components/Transcript.tsx

```diff
@@ -182,14 +184,14 @@ function TranscriptSegment({
       />
       <p
         className={classnames(
-          'grow text-justify peer-hover:text-stone-900',
+          'grow peer-hover:text-stone-900',
```

Comment:
Justification is a tricky beast (this is my typesetting days coming through). You have to have a reliable character length to avoid big white rivers of space, and though justified text is sometimes aesthetically satisfying for the designer because of its tight edges, it's actually easier to read ragged-right text (in left-to-right languages, anyway). When working with the layout, I spotted some lines that were getting gappy and awkward.

I can cite references if you'd like me to take the time to do so!